login-form/src/components/page-components/Dashboard.js

```diff
@@ -1,11 +1,17 @@
 import React, { useEffect } from "react";
 import { connect } from "react-redux";
 import { Box } from "rebass";
-
 import { getDashboardData } from "../../redux/actions";
 import { RebassHeading } from "../ui-components/RebassHeading";
 import { RebassLabel } from "../ui-components/RebassLabel";
 import { LoadingScreen } from "./LoadingScreen";
+import {
+  reselectEmail,
```

Comment:
```suggestion
  selectEmail,
```

even though you are using reselect module this is still a selector. This may be opinionated but I prefer `select` as a prefix.